api/v1alpha1/helpers.go

```diff
@@ -33,14 +33,6 @@ func AddFinalizer(ctx context.Context, c client.Client, obj client.Object) error
 	return nil
 }
 
-func ClearAnnotations(ctx context.Context, c client.Client, obj client.Object, keys ...string) error {
-	kvs := make(map[string]*string, len(keys))
-	for _, k := range keys {
-		kvs[k] = nil
-	}
-	return patchAnnotations(ctx, c, obj, kvs)
-}
```

Comment:
Seems we stopped clearing refresh annotation quite some time ago and that was the only thing we were ever using this for.